sdk/javascript/client/deferred.js

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2016 Intel Corporation
+ * Copyright 2017 Intel Corporation
```

Comment:
This looks like a git rename, but the only thing that is the same in that file is the copyright header.  Everything else is pretty much new.